app/graphql/types/edition_type.rb

```diff
@@ -2,6 +2,35 @@
 
 module Types
   class EditionType < Types::BaseObject
-    field :title, String
+    field :analytics_identifier, String
+    field :base_path, String
+    field :content_id, ID
+    field :description, String
+    field :details, GraphQL::Types::JSON, null: false
+    field :document_type, String
+    field :first_published_at, GraphQL::Types::ISO8601DateTime, null: false
+    # TODO: work out how to retrieve the links returned in the content item
+    field :links, [GraphQL::Types::JSON], null: false
```

Comment:
Perhaps we should just remove links for now, as they aren't yet properly supported by this endpoint?

We may choose to present links in a completely different way once we get to other pages (e.g. the ministers index page) so a placeholder may not immediately be useful.